projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.ts

```diff
@@ -0,0 +1,120 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { DateCoercer, Dictionary } from '@hypertrace/common';
+import {
+  CoreTableCellRendererType,
+  ListViewHeader,
+  ListViewRecord,
+  TableColumnConfig,
+  TableDataResponse,
+  TableDataSource,
+  TableMode,
+  TableRow
+} from '@hypertrace/components';
+import { isEmpty } from 'lodash-es';
+import { Observable, of } from 'rxjs';
+
+export const enum LogEventsTableViewType {
+  Sheet = 'sheet',
```

Comment:
We do need better name for this. Umm. How about `Condensed` for the sheet and `Detailed` for the page? Any other suggestions anyone?